_includes/default-tutorial.html

```diff
@@ -0,0 +1,476 @@
+<h1>Welcome</h1>
+
+<p>Welcome to AllenNLP! This tutorial will walk you through the basics of building and training an AllenNLP model.</p>
+
+{% include more-tutorials.html %}
+
+<p>Before we get started, make sure you have a clean Python 3.6 or 3.7 virtual environment, and then run the following command to install the AllenNLP library:</p>
+
+{% highlight bash %}
+pip install allennlp
+{% endhighlight %}
+
+<hr />
+
+<p>In this tutorial we'll implement a slightly enhanced version of the PyTorch
+<a href = "https://pytorch.org/tutorials/beginner/nlp/sequence_models_tutorial.html#example-an-lstm-for-part-of-speech-tagging">LSTM for Part-of-Speech Tagging</a> tutorial,
+adding some features that make it a slightly more realistic task (and that also showcase some of the benefits of AllenNLP):</p>
+
+<ol class="formatted">
+  <li>We'll read our data from files. (The tutorial example uses data that's given as part of the Python code.)</li>
+  <li>We'll use a separate validation dataset to check our performance. (The tutorial example trains and evaluates on the same dataset.)</li>
+  <li>We'll use <a href="https://github.com/tqdm/tqdm" target="_blank">tqdm</a> to track the progress of our training.</li>
+  <li>We'll implement <a href="https://en.wikipedia.org/wiki/Early_stopping" target="_blank">early stopping</a> based on the loss on the validation dataset.</li>
+  <li>We'll track accuracy on both the training and validation sets as we train the model.</li>
+</ol>
+
+<hr />
+
+<h2>The Problem</h2>
+
+<p>Given a sentence (e.g. <code>"The dog ate the apple"</code>) we want to predict part-of-speech tags for each word<br />(e.g <code>["DET", "NN", "V", "DET", "NN"]</code>).</p>
+
+<p>As in the PyTorch tutorial, we'll embed each word in a low-dimensional space, pass them through an LSTM to get a sequence of encodings, and use a feedforward layer to transform those into a sequence of logits (corresponding to the possible part-of-speech tags).</p>
+
+<p>Below is the annotated code for accomplishing this.</p>
+
+<!-- Annotated Code -->
```

Comment:
Code output should be structured like this. HTML comments (e.g. `<!-- ... -->`) are optional.